Spelling and grammar changes (#1301)
helium/docs: docs/vote-escrow/realms.mdx

### Duration

> The duration of your cooldown is defined as days. The longer your tokens are staked, the more veHNT

Reviewer: ```suggestion
The duration of your cooldown is set in units of days. The longer your tokens are staked, the more veHNT
```
How about something like this?

Author: Too wordy imo, most of the people reading probably fly through anyways.
